insuree/gql_queries.py

```diff
@@ -39,6 +40,17 @@ class Meta:
             "id": ["exact"]
         }
 
+class IncomeLevelsGQLType(DjangoObjectType):
+
+    class Meta:
+        model = IncomeLevels
+        interfaces = (graphene.relay.Node,)
+        filter_fields = {
+            "id":["exact"],
+            "french_version":["exact", "istartswith", "icontains", "iexact"],
```

Comment:
You should only match based on id and not the value. Check also the other comment on the linked migration file.